database/seeds/ModuleSeeder.php

```diff
@@ -13,39 +13,16 @@ class ModuleSeeder extends Seeder
      */
     public function run()
     {
+
         ModulePage::create([
-        	'name'			=>	'Antennae management',
-        	'code'			=>	'antennae_management',
-        	'module_link'	=>	'modules/loggedIn/antennae_management/antennae_management.js',
+        	'name'			=>	'Body management',
```

Comment:
Same as migrations. Do not modify existing seeds since they will NOT run on existing installations if they have already been ran.